places/tests.py

```diff
@@ -0,0 +1,89 @@
+from django.test import TestCase
+from . import mock
+import responses, requests, os
+
+API_KEY = os.getenv('API_KEY')
+Q = 'lagos'
+SOURCE = 'google'
+LNG = 3.3792057
+LAT = 6.5243793
+PLACE_ID = "ChIJwYCC5iqLOxARy9nDZ6OHntw"
+COMMON = "https://maps.googleapis.com/maps/api/place/"
+URL = f"{COMMON}findplacefromtext/json?inputtype=textquery&key={API_KEY}&input="
+ZERO_SEARCH_URL = f"{URL}xxxx"
+SEARCH_URL = f"{URL}{Q}"
+SEARCH_URL_WITH_LOCATION = f"{SEARCH_URL}&locationbias=point:{LNG},{LAT}"
+DETAILS_URL = f"{COMMON}details/json?place_id={PLACE_ID}&key={API_KEY}"
+DETAILS = mock.details
+NO_CANDIDATE = mock.no_candidate
+CANDIDATE = mock.candidate
+RESULT = {
+    'status': 'OK',
+    'result': {
+        "ID": "f5e45a1cb67f5581896d92388b6402c552e4d5f0", 
+        "Provider": "GOOGLE", 
+        "Name": "Lagos", 
+        "Location": {"Lat": LAT, "Lng": LNG}, 
+        "Address": "Lagos, Nigeria", 
+        "URI": "https://maps.google.com/?q=Lagos,+Nigeria&ftid=0x103b8b2ae68280c1:0xdc9e87a367c3d9cb"
+    }
+}
+ZERO_RESULT = {'status': 'ZERO_RESULTS', 'result': []}
+
+
+# Create your tests here.
+class SearchTest(TestCase):
```

Comment:
Shouldn't there be something like a setUp method here?

Reply:
Not needed